src/event/block/BlockSproutEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\event\block;
+
+use pocketmine\block\Block;
+use pocketmine\world\BlockTransaction;
+
+/**
+ * Called when Saplings or Bamboo grow.
+ * These types of plants tend to change multiple blocks at once upon growing, that's why this event returns the BlockTransaction.
+ */
+class BlockSproutEvent extends BaseBlockChangeEvent{
```

Comment:
This is misleading. It's not a single block that's being changed, while this extension (and thereby the presence of `::getBlock()` and `::getNewState()`) implies that it is.